src/api/interactivity.js

```diff
@@ -114,6 +115,7 @@ export default class Interactivity {
      */
     on(eventName, callback) {
         checkEvent(eventName);
+        this._numListeners[eventName] = (this._numListeners[eventName] || 0) + 1;
```

Comment:
I would suggest to initialize all the listeners to 0 in the constructor (since we only have five events we are listening) to avoid this check.